### PR TITLE
changes for IAM_APIKEY credentials

### PR DESCRIPTION
**Pull request by rhagarty**
- added env.sample file to make similar to other node patterns
- added IAM_APIKEY processing and instructions
- fixed npm package vulnerability issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -377,6 +379,32 @@ If the baseball season isn't over, make sure the following variable is set corre
...
+If you have run into the limit issue with your plan, simply delete and re-create a new instance of the service. And remember to update the .env file with your new credentials. 
+
+### When running on Mac OSX, microphone is not picking up any sound
+
+Ensure that the basic microphone functionality is working:
+
+```
+sox -d test.wav        // speak into mic, then ctrl-c to exit
+sox test.wav -d        // playback
+
```
Comment (stevemar) [inline README.md]: need to end the code block here

File: .env [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+# Watson Assistant authentication using username/password authentication
+CONVERSATION_USERNAME=f61d086f-92a0-415a-bf6e-a6c278e979bc
+CONVERSATION_PASSWORD=34QeMuxEXlbx
+# Watson Assistant authentication using IAM
+#CONVERSATION_IAM_APIKEY=<put assistant IAM apikey here>
+#CONVERSATION_URL=<put assistant url here>
+CONVERSATION_WORKSPACE_ID=ba98c98e-d04f-4ad1-a62c-b93791f66100
+
+# Watson Tone Analyzer authentication using username/password authentication
+TONE_ANALYZER_USERNAME = 2a7edb09-5a4d-428f-ad13-0732e325933a
```
Comment (stevemar) [inline .env]: consider deleting your keys before checking these in...

**Review by stevemar — COMMENTED** [review]
nice job

**Comment by stevemar** [comment]
i didn't approve because it's marked WIP, when you're ready for it to merge, let me know.

**Review by tanmayb123 — APPROVED** [review]

**Pull request merged**